### PR TITLE
Moved text from :before to <button>

### PR DESCRIPTION
**Pull request by vinbaluyot**
Moved the button's text from `:before`'s `content` property to inside of the `<button>`.
_Side note: Please see my comment below regarding the compiler._

Issue: #803

### CODE REVIEW

**Comment by shahednasser** [comment]
Hi @vinbaluyot there's no problem with the compiler, it just compiles everything (not based on what changed).

Can you please commit the changes after compiling? Changes in `dist/sbuttons.css` are necessary.

**Comment by vinbaluyot** [comment]
> Hi @vinbaluyot there's no problem with the compiler, it just compiles everything (not based on what changed).
> 
> Can you please commit the changes after compiling? Changes in `dist/sbuttons.css` are necessary.

@shahednasser  It's done and I think I did it correctly this time. And of course `dist/sbuttons.css` is necessary. Silly me lol.

**Review by shahednasser — CHANGES_REQUESTED** [review]
It looks great just a few remarks:

1. Include a rule for `.stack-btn.block-btn` where display is flex. I think you will need to add `!important` to work.
2. Change the margin for `.stack-btn.block-btn` to `2rem auto`

**Comment by vinbaluyot** [comment]
> It looks great just a few remarks:
> 
> 1. Include a rule for `.stack-btn.block-btn` where display is flex. I think you will need to add `!important` to work.
> 2. Change the margin for `.stack-btn.block-btn` to `2rem auto`

To avoid using `!important`, I modified `.stack-btn`'s display from `flex` to `inline-flex` and added `display: flex` to `.stack-btn.block-btn` (also changed the margin as you mentioned above). Hope that works for you.

File: src/components/styles/_stack.less [hunk 1]
```diff
@@ -86,7 +103,8 @@
   padding: 10px 0 20px 0;
   height: 90%;
   width: 90%;
-  display: flex;
+  // display: flex;
```
Comment (shahednasser) [inline src/components/styles/_stack.less:106]: Ok all looks great just remove this comment

Comment (vinbaluyot) [reply src/components/styles/_stack.less]: Ok the comment has been removed :)

**Comment by shahednasser** [comment]
Great work thank you!

@all-contributors please add @vinbaluyot for code and bug

**Pull request merged**